src/js/common/utils/static-page-widgets.js

```diff
@@ -0,0 +1,42 @@
+function mountWidgets(widgets) {
```

Comment:
The point of this file is to handle failing gracefully when something goes wrong with the JS, and to make sure we're showing a spinner while JS is loading.

Reply:
Would ☝️  make a useful comment in the code for later?

Reply:
I was trying to avoid comments in here, since they won't get stripped out.

Reply:
Ohh, I see. Hmm.